pkg/resource/image_metadata.go

```diff
@@ -34,15 +34,17 @@ import (
 const (
 	// DefaultExportedPort is the default protocol exposed by inner services specified in image metadata
 	DefaultExportedPort = "http"
-	// LabelKeyPrometheus is the label key for Prometheus metadata
-	LabelKeyPrometheus = "prometheus.io"
 	// LabelKeyOrgKie is the label key for KIE metadata
 	LabelKeyOrgKie = "org.kie" + labelNamespaceSep
 	// LabelKeyOrgKiePersistence is the label key for Persistence metadata
-	LabelKeyOrgKiePersistence = "org.kie" + labelNamespaceSep + "persistence"
+	LabelKeyOrgKiePersistence = LabelKeyOrgKie + "persistence"
```

Comment:
is it missing a "." before persistence?

Reply:
it's not. The separator is there: `\`, named `labelNamespaceSep`. There's also a unit test to verify it.